config/identity/config.yaml

```diff
@@ -120,45 +124,78 @@ meta-issuers:
     ci-provider: *github-type
 ci-issuer-metadata:
   *github-type:
-      default-template-values:
-          url: "https://github.com"
-      extension-templates:
-          github-workflow-trigger: "event_name"
-          github-workflow-sha: "sha"
-          github-workflow-name: "workflow"
-          github-workflow-repository: "repository"
-          github-workflow-ref: "ref"
-          build-signer-uri: "{{ .url }}/{{ .job_workflow_ref }}"
-          build-signer-digest: "job_workflow_sha"
-          runner-environment: "runner_environment"
-          source-repository-uri: "{{ .url }}/{{ .repository }}"
-          source-repository-digest: "sha"
-          source-repository-ref: "ref"
-          source-repository-identifier: "repository_id"
-          source-repository-owner-uri: "{{ .url }}/{{ .repository_owner }}"
-          source-repository-owner-identifier: "repository_owner_id"
-          build-config-uri: "{{ .url }}/{{ .workflow_ref }}"
-          build-config-digest: "workflow_sha"
-          build-trigger: "event_name"
-          run-invocation-uri: "{{ .url }}/{{ .repository }}/actions/runs/{{ .run_id }}/attempts/{{ .run_attempt }}"
-          source-repository-visibility-at-signing: "repository_visibility"
-      subject-alternative-name-template: "{{ .url }}/{{ .job_workflow_ref }}"
+    default-template-values:
+      url: "https://github.com"
+    extension-templates:
+      github-workflow-trigger: "event_name"
```

Comment:
While we are updating this, can you add the code comments for each of these claims from the GitHub and gitlab principals?